haskell-src/Concordium/Types.hs

```diff
@@ -711,7 +711,23 @@ instance Show Address where
 
 -- | Time in seconds since the unix epoch
 newtype TransactionTime = TransactionTime {ttsSeconds :: Word64}
-    deriving (Show, Read, Eq, Num, Ord, FromJSON, ToJSON, Real, Enum, Integral) via Word64
+    deriving (Show, Read, Eq, Num, Ord, Real, Enum, Integral) via Word64
```

Comment:
**Potential issue:** This library is used by both the `concordium-node` and the `concordium-wallet-proxy`, so we need to ensure this is not breaking anything. 
Don't think anything in the node depends on the JSON format anymore, but I imagine this is exposed by the wallet-proxy.

Reply:
I kept the old `fromJSON`/`toJSON` implementation now.